tests/test_geom_opt.py

```diff
@@ -187,3 +192,22 @@ def test_restart(tmp_path):
     )
     final_energy = single_point.run("energy")["energy"]
     assert final_energy < intermediate_energy
+
+
+def test_space_group():
+    """Test spacegroup of the  structure."""
+
+    single_point = SinglePoint(
+        struct_path=DATA_PATH / "NaCl-sg.cif",
+        architecture="mace_mp",
+        calc_kwargs={"model": MODEL_PATH},
+    )
+
+    optimize(
+        single_point.struct,
+        fmax=0.001,
+        filter_func=DefaultFilter,
```

Comment:
Isn't this (with the `try...except` import changes) duplicating what is already done in the optimize module?